### PR TITLE
added esdt transfer and send functions for any ESDT

### PR DESCRIPTION
**Pull request by catalinnnn**
Identified this as being a case where a lot of contracts might do the handle independently, each on their own.
Think it's best we do the handling in the framework.

### CODE REVIEW

File: elrond-wasm/src/api/send_api.rs [hunk 1]
```diff
@@ -76,6 +76,24 @@ where
 		}
 	}
 
+	/// Sends ESDT tokens to the target address. Handles any type of ESDT.
+	fn transfer_esdt(&self, token: &TokenIdentifier, nonce: u64, amount: &BigUint, to: &Address) {
```
Comment (claudiulataretu) [inline elrond-wasm/src/api/send_api.rs:80]: I would rather call this method transfer_tokens. You are handling both ESDTs and NFTs in the same method

File: elrond-wasm/src/api/send_api.rs [hunk 2]
```diff
@@ -277,6 +295,17 @@ where
 		self.call_local_esdt_built_in_function(gas, b"ESDTNFTBurn", &arg_buffer);
 	}
 
+	/// Burns ESDT tokens. Handles any type of ESDT.
+	fn burn_esdt(&self, token: &TokenIdentifier, nonce: u64, amount: &BigUint, gas: u64) {
```
Comment (claudiulataretu) [inline elrond-wasm/src/api/send_api.rs:299]: I would rather call this method burn_tokens. You are handling both ESDTs and NFTs in the same method

**Review by claudiulataretu — CHANGES_REQUESTED** [review]

File: elrond-wasm/src/api/send_api.rs (hunk 1 above)
Comment (andrei-marinica) [reply elrond-wasm/src/api/send_api.rs:80]: `transfer_tokens` is better, yes

Comment (catalinnnn) [reply elrond-wasm/src/api/send_api.rs:80]: NFTs are ESDTs. transfer_tokens means any tokens, including EGLD. this only works with ESDT, hence the naming

**Review by dorin-iancu — APPROVED** [review]

**Review by andrei-marinica — APPROVED** [review]

**Review by claudiulataretu — APPROVED** [review]

**Pull request merged**